pinecone/sparse/splade/splade_quora.ipynb

```diff
@@ -0,0 +1,2438 @@
+{
```

Comment:
<blockquote>Here we define a function that merges the query results with the actual texts of the documents and shows them as a dataframe.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>